lib/logger.js

```diff
@@ -70,20 +73,35 @@ function now () {
 }
 
 function createLogger (options) {
-  if (!options.logger) {
+  // if no logger is provided, then create a default logger
+  if (!options.loggerInstance && !options.logger) {
     const logger = nullLogger
     logger.child = () => logger
     return { logger, hasLogger: false }
   }
 
-  if (validateLogger(options.logger)) {
+  if (options.logger && options.loggerInstance) {
+    throw new FST_ERR_LOGGER_AND_LOGGER_INSTANCE_PROVIDED()
```

Comment:
My bad. If I see the other Errors, than it should have been FST_ERR_LOG_ prefixed